ols/user_data_collection/data_collector.py

```diff
@@ -257,14 +257,18 @@ def upload_data_to_ingress(tarball: io.BytesIO) -> requests.Response:
             "Authorization": f"Bearer {token}",
         }
 
-    with requests.Session() as s:
-        s.headers = headers
-        logger.debug(f"posting payload to {url}")
-        response = s.post(
-            url=url,
-            files=payload,
-            timeout=INGRESS_TIMEOUT,
-        )
+    try:
+        with requests.Session() as s:
+            s.headers = headers
+            logger.debug(f"posting payload to {url}")
+            response = s.post(
+                url=url,
+                files=payload,
+                timeout=INGRESS_TIMEOUT,
+            )
+    except json.JSONDecodeError:
+        logger.error(f"can't decode JSON content, response: {response}")
```

Comment:
please put a little more context in this error message about what OLS was trying to do/what failed.  Keep in mind this might be the only message a user sees in the log and they will have no idea what the implication is when they see it.